Symfony/CS/Tests/IntegrationTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests;
+
+/**
+ * Test that parses and runs the fixture *.test files found in
+ * /Fixtures/Integration.
+ *
+ * @author SpacePossum
+ */
+class IntegrationTest extends AbstractIntegrationTest
```

Comment:
Not easier, the Integration test is not meant to be extended. That way it can still be changed within the PHP-CS-Fixer repo without being a BC-break, maybe I should make it final...